api/v2/app/auth/helpers.py

```diff
@@ -0,0 +1,72 @@
+import re
+from werkzeug.security import generate_password_hash, check_password_hash
+from functools import wraps
+from flask import request, jsonify, make_response
+
+from app.models import User
+
+
+def email_validator(email):
+    '''validates user provided email'''
+    if re.match(
+            "^[_a-zA-Z0-9-]+(\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+(\.[a-zA-Z0-9-]+)*(\.[a-zA-Z]{2,4})$",
+            email):
+        return True
+
+
+def address_validator(address):
+    '''Handles the validation of address'''
+    if re.match(
+            '^[#.0-9a-zA-Z\s,-]+$',
+            address):
+        return True
+
+
+def password_validator(password):
+    '''validates user provided password length'''
+    if len(password) > 6:
```

Comment:
I expected checks for other password formats apart from length. But either way, still cool